internal/jobcontainers/utils.go

```diff
@@ -0,0 +1,25 @@
+package jobcontainers
+
+import (
+	"strings"
+
+	"github.com/Microsoft/hcsshim/internal/jobobject"
+)
+
+// Seperates path to executable/cmd from it's arguments
+func seperateArgs(cmdline string) (string, []string) {
+	split := strings.Fields(cmdline)
+	return split[0], split[1:]
```

Comment:
There should probably be some error checking here so we don't panic when the cmdline is just the executable

Reply:
It can be just the executable, that's generally all I used to run for testing things was just "/some/binary.exe", [1:] works fine with one element in a slice.

Reply:
:0 you're right, I didn't even know that